gwpy/timeseries/timeseries.py

```diff
@@ -1805,13 +1920,17 @@ def q_transform(self, qrange=(4, 64), frange=(0, numpy.inf),
             highpass corner frequency (in Hz) of the FIR whitening filter,
             used only if `whiten` is not `False`, default: `None`
 
+        qgram : `~gwpy.signal.qtransform.QPlane`, optional
+            ready-made time-frequency plane (at fixed Q) over which to
+            interpolate, will override `whiten` if passed, default: `None`
+
```

Comment:
This keyword really doesn't belong here, since its inclusion removes all usage of the parent `TimeSeries` object in the method and reduces it to an alias of `qgram.interpolate`. If the user already has a `<qgram>` object, they can call the `interpolate` method themselves?